<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatershedClass</class>
 <widget class="QMainWindow" name="WatershedClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pandore Watershed Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Watershed options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Show</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QRadioButton" name="optShowOryginalImage">
                <property name="text">
                 <string>Oryginal image</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="bttBlurOryginalImage">
                <property name="minimumSize">
                 <size>
                  <width>82</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Blur</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="optShowImageForComputations">
              <property name="text">
               <string>Image for computations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="optShowInitialImage">
              <property name="text">
               <string>Initial image</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Set image for computations</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QRadioButton" name="optSetGradientImage">
                <property name="text">
                 <string>Gradient image</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="optSetDistanceMap">
                <property name="text">
                 <string>Distance map</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="bttComputeGradient">
                <property name="text">
                 <string>Compute</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bttComputeDistanceMap">
                <property name="text">
                 <string>Compute</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Set initial image</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="optSetLocalMinimum">
                <property name="text">
                 <string>Local minimum point</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="optSetBinaryImage">
                <property name="text">
                 <string>Binary image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="optSetMarker">
                <property name="text">
                 <string>Marker</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="bttClearMarkers">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QPushButton" name="bttComputeLocalMinimumPoints">
                <property name="text">
                 <string>Compute</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bttComputeBinaryImage">
                <property name="text">
                 <string>Compute</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bttDrawMarkers">
                <property name="toolTip">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use &lt;span style=&quot; text-decoration: underline;&quot;&gt;left&lt;/span&gt; mouse button to &lt;span style=&quot; font-weight:600;&quot;&gt;draw&lt;/span&gt;. &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use &lt;span style=&quot; text-decoration: underline;&quot;&gt;right&lt;/span&gt; mouse button to &lt;span style=&quot; font-weight:600;&quot;&gt;erase&lt;/span&gt;. &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use mause wheel to change drawing tool size.&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Draw</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bttRunWatershed">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>520</width>
         <height>533</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="ImageCanvas" name="imageLabel">
          <property name="text">
           <string>Open image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_oryginal_image"/>
    <addaction name="actionOpen_gradient_image"/>
    <addaction name="actionOpen_distance_map"/>
    <addaction name="actionOpen_marker_image"/>
    <addaction name="separator"/>
    <addaction name="actionSave_result"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Pandore"/>
    <addaction name="actionAbout_this_GUI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_oryginal_image">
   <property name="text">
    <string>Open oryginal image</string>
   </property>
  </action>
  <action name="actionOpen_marker_image">
   <property name="text">
    <string>Open marker image</string>
   </property>
  </action>
  <action name="actionOpen_gradient_image">
   <property name="text">
    <string>Open gradient image</string>
   </property>
  </action>
  <action name="actionOpen_distance_map">
   <property name="text">
    <string>Open distance map</string>
   </property>
  </action>
  <action name="actionLoad_test_data">
   <property name="text">
    <string>Load test images</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_image_for_computation">
   <property name="text">
    <string>Open image for computation</string>
   </property>
  </action>
  <action name="actionAbout_Pandore">
   <property name="text">
    <string>About Pandore</string>
   </property>
  </action>
  <action name="actionAbout_this_GUI">
   <property name="text">
    <string>About this GUI</string>
   </property>
  </action>
  <action name="actionSave_result">
   <property name="text">
    <string>Save result</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageCanvas</class>
   <extends>QLabel</extends>
   <header>imagecanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>optSetGradientImage</sender>
   <signal>clicked(bool)</signal>
   <receiver>optShowImageForComputations</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optSetDistanceMap</sender>
   <signal>toggled(bool)</signal>
   <receiver>optShowImageForComputations</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optSetLocalMinimum</sender>
   <signal>clicked(bool)</signal>
   <receiver>optShowInitialImage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optSetMarker</sender>
   <signal>toggled(bool)</signal>
   <receiver>optShowInitialImage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>WatershedClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optSetBinaryImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>optShowInitialImage</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttComputeGradient</sender>
   <signal>pressed()</signal>
   <receiver>optSetGradientImage</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttComputeDistanceMap</sender>
   <signal>pressed()</signal>
   <receiver>optSetDistanceMap</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttBlurOryginalImage</sender>
   <signal>pressed()</signal>
   <receiver>optShowOryginalImage</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttComputeLocalMinimumPoints</sender>
   <signal>pressed()</signal>
   <receiver>optSetLocalMinimum</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttComputeBinaryImage</sender>
   <signal>pressed()</signal>
   <receiver>optSetBinaryImage</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttDrawMarkers</sender>
   <signal>pressed()</signal>
   <receiver>optSetMarker</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttComputeGradient</sender>
   <signal>pressed()</signal>
   <receiver>optShowImageForComputations</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttComputeLocalMinimumPoints</sender>
   <signal>pressed()</signal>
   <receiver>optShowInitialImage</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
